corehq/apps/sms/migrations/0059_remove_unicel_backend.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.2.14 on 2024-07-26 10:39
+
+from django.db import migrations
+
+
+def remove_unicel_backend(apps, schema_editor):
+    SQLMobileBackend = apps.get_model('sms', 'SQLMobileBackend')
+    unicel_backends = SQLMobileBackend.objects.filter(hq_api_id='UNICEL')
+    unicel_backends.delete()
```

Comment:
Do you know if any other models reference these rows (via foreign key references, for example)? Do we need to worry about those references being broken (possibly causing errors) or this delete command failing as a result of those references?

Another possibility is that this delete will _cascade_ to rows in other tables (like SMS records). It would be a bad thing if SMS history were removed as a result of deleting the gateway, and if that is the case then we should find a way to avoid that. We should verify that historical SMS records that originated from the Unicel gateway retain enough context to be useful.